Sources/Apodini/WebService.swift

```diff
@@ -78,9 +79,14 @@ extension WebService {
         try Self.start(mode: .run, webService: self)
     }
     
-    /// The command configuration of the `ParsableCommand`
+    /// The command configuration of the `ParsableCommand`.
+    /// The default value depends if there are arguments or options specified in the web service.
+    /// If so, it contains the default empty configuration.
+    /// If not, it automatically adds the commands of the specified `Configurations` as sub command.
     public static var configuration: CommandConfiguration {
-        CommandConfiguration(subcommands: Self().configuration._commands)
+        let mirror = Mirror(reflecting: Self())
+        let hasNoRelevantChildren = !mirror.children.contains { $0.value is CLIParsable }
+        return CommandConfiguration(subcommands: hasNoRelevantChildren ? Self().configuration._commands : [])
```

Comment:
I took a short look at the problem and I still think the solution you proposed here is problematic @hendesi:
As noted by @philippzagar we are missing some possible property wrappers and I think this already highlights the problem with the approach. The Swift argument parser provides a `parseOrExit` method that can be loaded from the globally available `CommandLine.arguments`.

So if a `WebService` has default properties for all argument parser property wrappers this works like a charm and if all arguments are passed in that are required this should also work as expected. In addition this approach doesn't override the static configuration but just appends some additional subcommands:  
```suggestion
        let subcommands = Self.parseOrExit(CommandLine.arguments).configuration._commands
        var configuration = Self.configuration
        configuration.subcommands.append(contentsOf: subcommands)
        return configuration
```

I think this also addresses a Problem with the Deployment provider CLI. 

In addition: A deployment provider **must** have the possibility to pass any arbitrary arguments to a sort of "passthrough arguments" argument that are just piped through to the web service. E.g. if the web service as an argument that must be set the arguments are piped through to the web service. 
I think we should merge this PR as soon as possible so adding this to the deployment providers should be done later in an other PR but IMO this should also be done sooner than later.

Reply:
@hendesi I have updated the comment above with a longer explanation and have pushed my ideas to an other branch. I will update the comment here after creating the other PR.